test/monitoring/config-grafana/provisioning/dashboards-data/storage_experiments.json

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
Is the idea to eventually rename this to `storage` once it's past `experiments` stage?

Reply:
Yes! Currently, it has metrics that were not introduced on `main` yet and also some metrics that Jasmina uses for the current storage experiments but they may not be introduced in a release at all. (Honestly, I released this because I had to share the storage board with her.)

Actually, we can rename it right now, so we don't forget. But the content might change in the future.